docs/themes/v2/source/css/styles.css

```diff
@@ -1085,6 +1087,13 @@ main {
   --color-background: var(--color-yellow-5);
 }
 
+.content-markdown .admonition.info {
+  --color-border: #DD7DFF;
+  --color-title: var(--color-white);
+  --color-title-background: #DD7DFF;
+  --color-background: #FDF6FF;
```

Comment:
Could we reuse the Purple (Plum) 0 here? The shade is slightly different `#FBF2FC` but it should work

```suggestion
  --color-background: var(--color-purple-0);
```

Reply:
Sounds good